SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2258,6 +2258,9 @@ void USpatialNetDriver::HandleStartupOpQueueing(const TArray<Worker_OpList*>& In
 
 		if (bIsReadyToStart)
 		{
+			// We know at this point that we have all the information to set the worker's interest query.
+			Sender->SendServerWorkerEntityInterestUpdate();
```

Comment:
@alastairdglennie settled on this point for the "we can definitely set this correctly", lmk if this is not what you had in mind

Reply:
Yep this is where I thought it should go. I think we should gate this for when load balancing is enabled

Reply:
done